### PR TITLE
GCP-8 Button presets

### PR DESCRIPTION
**Pull request by patilvikasj**
### Checklist:
- [ ] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

**Comment by Nikschavan** [comment]
@patilvikasj should this Pr be against the `next-release` branch?

**Comment by patilvikasj** [comment]
> @patilvikasj should this Pr be against the `next-release` branch?

@Nikschavan Actually, this will be released as a separate update. So probably it will be first merged to milestones ( Or a new branch ) first and then master.

File: inc/customizer/extend-custom-controls/src/slider/slider-component.js [hunk 1]
```diff
@@ -86,4 +103,4 @@ SliderComponent.propTypes = {
 	control: PropTypes.object.isRequired
 };
 
-export default React.memo( SliderComponent );
+export default SliderComponent;
```
Comment (premanshup) [inline inc/customizer/extend-custom-controls/src/slider/slider-component.js:106]:
```suggestion
export default React.memo( SliderComponent );
```

File: inc/customizer/extend-custom-controls/src/responsive-slider/responsive-slider-component.js [hunk 2]
```diff
@@ -9,6 +9,8 @@ const ResponsiveSliderComponent = props => {
 	const [state, setState] = useState( prop_value );
 
 	useEffect( () => {
+
+		console.log( "inside useeffect" );
```
Comment (premanshup) [inline inc/customizer/extend-custom-controls/src/responsive-slider/responsive-slider-component.js:13]:
```suggestion

```

**Pull request closed without merging**